src/web/ng/impl/WsConnection.hpp

```diff
@@ -112,20 +125,29 @@ class WsConnection : public Connection {
     }
 
     std::optional<Error>
-    send(
-        Response response,
+    sendBuffer(
+        boost::asio::const_buffer buffer,
         boost::asio::yield_context yield,
-        std::chrono::steady_clock::duration timeout = DEFAULT_TIMEOUT
+        std::chrono::steady_clock::duration timeout = Connection::DEFAULT_TIMEOUT
     ) override
     {
-        auto error = util::withTimeout(
-            [this, &response](auto&& yield) { stream_.async_write(response.asConstBuffer(), yield); }, yield, timeout
-        );
+        auto error =
```

Comment:
I saw there is no longer a queue here. Do the multiple async_write overlap?
From the [document](https://boost.org/doc/libs/1_80_0/doc/html/boost_asio/reference/async_write/overload1.html)
```
The program must ensure that the stream performs no other write operations (such as async_write, the stream's async_write_some function, or any other composed operations that perform writes) until this operation completes.
``` 

Is it a problem?

Reply:
Connection is processed in a coroutine and each coroutine has its own strand, so it is fine here.